jenkins/stretch-ltp.jpl

```diff
@@ -0,0 +1,13 @@
+@Library('kernelci') _
+import org.kernelci.debian.RootFS
+
+def r = new RootFS()
+
+def config = ['name':"stretch-ltp",
+              'vm_memory':"3G",
+              'arch_list':["armhf", "arm64", "i386", "amd64"],
+              'debian_release':"stretch",
+              'extra_packages':"acl attr cpio cron curl expect file gawk gzip iproute2 libaio1 libc-bin logrotate net-tools perl procps python-all quota systemd tar unzip util-linux ",
+              'script':"scripts/stretch-ltp.sh"]
```

Comment:
Cosmetics: it's a bit better to add a trailing `,` and a new line, to reduce diff noise when adding things at the end.

Reply:
fixed